F/flex/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "flex"
+version = v"2.6.4"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/westes/flex/releases/download/v$(version)/flex-$(version).tar.gz",
+                  "e87aae032bf07c26f85ac0ed3250998c37621d95f8bd748b31f15b33c45ee995")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+apk add gettext-dev
+cd ${WORKSPACE}/srcdir/flex-*
+./autogen.sh
```

Comment:
Ok, the problem with `autopoint` from `Gettext_jll` is that it references `/workspace/destdir/share`, which doesn't exist when this is a host dependency.  `autopoint` is called during `./autogen.sh`, but why do we need to call it?  `configure` is already there
```suggestion
cd ${WORKSPACE}/srcdir/flex-*
```

Reply:
I'm mindlessly following the build instructions. Let's see what happens.